### PR TITLE
Security tool

### PR DESCRIPTION
**Pull request by pikajude**
Just want to get this PR open, there is some extra stuff that doesn't need to be in it that I will be removing

@copumpkin @jwiegley

### CODE REVIEW

File: pkgs/os-specific/darwin/apple-source-releases/libsecurity_generic/impure_deps.nix [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# generated using a ruby script
```
Comment (copumpkin) [inline pkgs/os-specific/darwin/apple-source-releases/libsecurity_generic/impure_deps.nix]: Is this specific to 10.10? This seems relevant to @shlevy's current interests

Comment (shlevy) [inline pkgs/os-specific/darwin/apple-source-releases/libsecurity_generic/impure_deps.nix]: Yeah, is the script available somewhere?

Comment (joelteon) [inline pkgs/os-specific/darwin/apple-source-releases/libsecurity_generic/impure_deps.nix]: https://gist.github.com/joelteon/40cde988dbbd5fc17cea

**Comment by joelteon** [comment]
@copumpkin ready

**Pull request closed without merging**